src/main/groovy/io/openliberty/tools/gradle/tasks/CompileJSPTask.groovy

```diff
@@ -15,21 +15,11 @@
  */
 package io.openliberty.tools.gradle.tasks
 
-import java.io.File;
-import java.text.MessageFormat;
-import java.util.List;
-import java.util.Set;
-import java.util.TreeSet;
-import java.util.HashSet;
-
-import org.gradle.api.tasks.TaskAction
+import io.openliberty.tools.ant.jsp.CompileJSPs
+import org.apache.tools.ant.Project
 import org.gradle.api.Task
-import org.gradle.api.GradleException
+import org.gradle.api.tasks.TaskAction
 import org.gradle.api.tasks.bundling.War
-import org.gradle.api.logging.LogLevel
-
-import org.apache.tools.ant.Project;
```

Comment:
Why are you removing all of these imports? Seems like it would cause the Project for the `ant` variable to use the wrong Project (Gradle one instead).

Reply:
k..I see the ant one I was worried about is there. But having the IDE remove imports for Groovy might be risky. Because Groovy doesn't complain until you actually run the code (and have all paths exercised). I see List, Set, HashSet, File all referenced in this groovy file. Removing those imports doesn't make sense to me.

Reply:
Build is successful after the reimports